proto/entities/v1alpha/entities.proto

```diff
@@ -108,3 +113,32 @@ message AuthorVersion {
   // For compound versions the greatest timestamp is used.
   google.protobuf.Timestamp version_time = 4;
 }
+
+// A change to an entity.
+message Entity {
+  // EID of the entity.
+  string id = 1;
+
+  // Title of the entity, depending onf the type:
+  // Alias in the case of account. 
+  // Title in the case of groups and documents 
+  // Empty in the case of comments.
+  string title = 2;
+}
+
+// Request to 
+message SearchLocalEntitesRequest {
+  // Title of the entities to look for. Since we use 
+  // Fuzzy search, a single title may return multiple 
+  // entities.
+  string title = 1;
```

Comment:
Maybe this should be called something more generic. Like “term” or “query”, because eventually we might want to support searching for other things, like actual content of the document, and so on.